specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/examples/CostAllocationTags.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "api-version": "2018-03-31",
+    "billingAccountId": "12345"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "providers/Microsoft.CostManagement/billingAccounts/{billingaccount-id}/providers/Microsoft.Consumption/costAllocationTags/costAllocationTags1",
+        "name": "costAllocationTags1",
+        "type": "Microsoft.Consumption/costAllocationTags",
+        "eTag": "\"1d34d012214157f\"",
```

Comment:
@asarkar84 - As per the definition of ["eTag" in the spec](https://github.com/asarkar84/azure-rest-api-specs/blob/def63250017ef0071aadd68a751623913f788af3/specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json#L2279), it is a string.

What is the reason behind escaping the quotes `"\"1d34d012214157f\""`? Wouldn't this `"1d34d012214157f"` be sufficient? Why is the service sending a string with quotes in it ?
/cc @sergey-shandar - FYI.

Reply:
I thought this is how azure services do eTag: " For example, I checked the CosmosDB data explorer which is publicly available. I found _eTag property exposed there and the service returns it with escaped quotes. Storage also has escaping quotes in their eTag.

PFB the CosmosDB example for reference:
_etag": "\"00000e3a-0000-0000-0000-59c94cd20000\"",

Reply:
@amarzavery Just refer any public API for reference, eTag will have quote. Escape char is used as per json standard since it will not accept 2 quotes like this ""

Reply:
Thank you for the explanation. That makes sense.